perma_web/api/views.py

```diff
@@ -582,7 +582,13 @@ def post(self, request, format=None):
                     'verb': 'GET'
                 }]
                 response = dispatch_multiple_requests(request.user, call_for_fresh_serializer_data)
-                return Response(response[0]['data'], status=status.HTTP_201_CREATED)
+                data = response[0]['data'].copy()
+
+                # include remaining links in response
+                links_remaining = request.user.get_links_remaining()
+                data['links_remaining'] = links_remaining
```

Comment:
Total nit, probably not worth picking: is there a reason not to combine lines 588-589 into `links_remaining = data['links_remaining'] = request.user.get_links_remaining()`?

Reply:
Ah, right. Fixed in eebba1855151781d90fd3a4320450c70ff7c8525